packages/plus/src/components/SteppedListCard/SteppedListContainer.tsx

```diff
@@ -0,0 +1,125 @@
+import styled from '@emotion/styled'
+import { Button, Row, Stack, StepList, Text } from '@ultraviolet/ui'
+import { Children, useMemo, useState } from 'react'
+import type { ReactNode } from 'react'
+import { SteppedList } from './Step'
+import { SteppedListContent } from './SteppedListContent'
+import { Data } from './helper'
+
+const StyledHeader = styled(Row)`
+  padding-bottom: ${({ theme }) => theme.space['3']};
+`
+
+const Content = styled(Row)`
+  border: solid ${({ theme }) => theme.colors.neutral.border};
+  width: 100%;
+`
```

Comment:
This can probably be replace by a Card and a Row inside of it without any styled 😄 

If you can try to avoid using styled